server/mediaserver.go

```diff
@@ -116,6 +116,7 @@ type authWebhookResponse struct {
 		Height  int    `json:"height"`
 		Bitrate int    `json:"bitrate"`
 		FPS     uint   `json:"fps"`
+		FPSDen  uint   `json:"fps_den"`
```

Comment:
I think we've used a camelCase convention for the other JSON fields (`manifestID` and `streamKey`); `fpsDen` is probably more consistent?